Description: Disable time stamps in conffiles, to avoid ever-changing files in /etc.
Author: Joey Hess <joeyh@debian.org>
Bug: http://www.cups.org/str.php?L3067
Bug-Debian: http://bugs.debian.org/549673

Index: cups-1.6.0/scheduler/classes.c
===================================================================
--- cups-1.6.0.orig/scheduler/classes.c	2011-11-04 07:37:03.000000000 +0100
+++ cups-1.6.0/scheduler/classes.c	2012-07-27 17:21:35.790180739 +0200
@@ -698,7 +698,7 @@
   strftime(temp, sizeof(temp) - 1, "%Y-%m-%d %H:%M", curdate);
 
   cupsFilePuts(fp, "# Class configuration file for " CUPS_SVERSION "\n");
-  cupsFilePrintf(fp, "# Written by cupsd on %s\n", temp);
+  cupsFilePrintf(fp, "# Written by cupsd\n");
   cupsFilePuts(fp, "# DO NOT EDIT THIS FILE WHEN CUPSD IS RUNNING\n");
 
  /*
Index: cups-1.6.0/scheduler/job.c
===================================================================
--- cups-1.6.0.orig/scheduler/job.c	2012-07-27 17:05:05.000000000 +0200
+++ cups-1.6.0/scheduler/job.c	2012-07-27 17:21:35.794180740 +0200
@@ -2133,7 +2133,7 @@
   strftime(temp, sizeof(temp) - 1, "%Y-%m-%d %H:%M", curdate);
 
   cupsFilePuts(fp, "# Job cache file for " CUPS_SVERSION "\n");
-  cupsFilePrintf(fp, "# Written by cupsd on %s\n", temp);
+  cupsFilePrintf(fp, "# Written by cupsd\n", temp);
   cupsFilePrintf(fp, "NextJobId %d\n", NextJobId);
 
  /*
Index: cups-1.6.0/scheduler/printers.c
===================================================================
--- cups-1.6.0.orig/scheduler/printers.c	2012-05-23 03:19:11.000000000 +0200
+++ cups-1.6.0/scheduler/printers.c	2012-07-27 17:21:35.794180740 +0200
@@ -1425,7 +1425,7 @@
   strftime(temp, sizeof(temp) - 1, "%Y-%m-%d %H:%M", curdate);
 
   cupsFilePuts(fp, "# Printer configuration file for " CUPS_SVERSION "\n");
-  cupsFilePrintf(fp, "# Written by cupsd on %s\n", temp);
+  cupsFilePrintf(fp, "# Written by cupsd\n");
   cupsFilePuts(fp, "# DO NOT EDIT THIS FILE WHEN CUPSD IS RUNNING\n");
 
  /*
Index: cups-1.6.0/scheduler/subscriptions.c
===================================================================
--- cups-1.6.0.orig/scheduler/subscriptions.c	2012-02-11 23:10:54.000000000 +0100
+++ cups-1.6.0/scheduler/subscriptions.c	2012-07-27 17:21:35.794180740 +0200
@@ -1108,7 +1108,7 @@
   strftime(temp, sizeof(temp) - 1, "%Y-%m-%d %H:%M", curdate);
 
   cupsFilePuts(fp, "# Subscription configuration file for " CUPS_SVERSION "\n");
-  cupsFilePrintf(fp, "# Written by cupsd on %s\n", temp);
+  cupsFilePrintf(fp, "# Written by cupsd\n");
 
   cupsFilePrintf(fp, "NextSubscriptionId %d\n", NextSubscriptionId);
 
